src/evolutionaryAlgorithm/EvolutionSimulator.py

```diff
@@ -52,6 +53,14 @@ def run(self):
     def initialize_population(self) -> List[IndividualInterface]:
         return self.initializator.initialize()
 
+    def markPopulation(self, population: List[IndividualInterface]) -> List[float]:
+        fitness = []
+        for el in population:
+            i = 0
+        #    i = ocena(el)
+            fitness.append(i)
+        return fitness
+
```

Comment:
ocena powinna być atrybutem osobnika a nie osobnąlistą floatów

Reply:
to dodałem prowizorycznie żeby było pokazane że musimy to zrobić

Reply:
Po prostu dodaj atrybut do reprezentacji osobnika, robienie prowizorycznie powoduje chaos.